online_meetings/canvas_zoom_meetings.py

```diff
@@ -67,42 +72,61 @@ def get_zoom_json(self, page_num: int = 1) -> Dict:
             return zoom_json["result"]
         return None
 
+    def extract_from_js(self, key: str, text: str) -> Optional[str]:
```

Comment:
Not sure if this should be a method if it doesn't require `self`.

Reply:
Yeah, good point, I could use a @staticmethod annotation here.